src/main.rs

```diff
@@ -80,18 +80,9 @@ fn draw(app: &mut App, gfx: &mut Graphics, state: &mut State) {
 
     // Draw transparent PNG with RenderTexture twice
     {
-        let mut d1 = gfx.create_draw();
-        d1.set_size(TEX_W, TEX_H);
-        d1.clear(Color::TRANSPARENT);
-        d1.image(&state.textures[1])
-            .scale_from((TEX_W / 2.0, TEX_H / 2.0), (1.0, -1.0));
-        gfx.render_to(&state.rt1, &d1);
-
-        let mut d2 = gfx.create_draw();
-        d2.set_size(TEX_W, TEX_H);
+        let mut d2 = state.rt2.create_draw();
         d2.clear(Color::TRANSPARENT);
-        d2.image(&state.rt1)
-            .scale_from((TEX_W / 2.0, TEX_H / 2.0), (1.0, -1.0));
+        d2.image(&state.rt1).blend_mode(BlendMode::OVER);
```

Comment:
Same as before, we need to use color blend mode as Over using premultiplied alpha images.